exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StorageResources.java

```diff
@@ -232,6 +209,32 @@ private JsonResult message(String message, Object... args) {
     return new JsonResult(String.format(message, args));
   }
 
+  private boolean isSupported(String format) {
+    return JSON_FORMAT.equalsIgnoreCase(format) || HOCON_FORMAT.equalsIgnoreCase(format);
+  }
+
+  private List<PluginConfigWrapper> getConfigsFor(String pluginGroup) {
+    return StreamSupport.stream(
+        Spliterators.spliteratorUnknownSize(storage.getStore().getAll(), Spliterator.ORDERED), false)
+            .filter(byPluginGroup(pluginGroup))
+            .map(entry -> new PluginConfigWrapper(entry.getKey(), entry.getValue()))
+            .sorted(PLUGIN_COMPARATOR)
+            .collect(Collectors.toList());
+  }
+
+  private Predicate<Map.Entry<String, StoragePluginConfig>> byPluginGroup(String pluginGroup) {
+    switch (pluginGroup == null ? "" : pluginGroup.toLowerCase()) {
```

Comment:
Please use separate if for null case, maybe its not that compact but more visible:
```
if (pluginGroup == null) {
   return entry -> false;
}
```

Reply:
@vdiravka @aielchiieva  sorry, my fault :)